google/resource_sql_database_instance.go

```diff
@@ -48,6 +48,7 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 						"activation_policy": &schema.Schema{
 							Type:     schema.TypeString,
 							Optional: true,
+							Computed: true,
```

Comment:
It looks like we're setting a lot of values to `Computed`, which worries me a bit because of what happens when you remove a value in config to get the default behaviour; where the resource will keep the last-set value for the field. How come we've had to make so much of this computed?